test/integrations/destinations/criteo_audience/dataDelivery/oauth.ts

```diff
@@ -0,0 +1,176 @@
+import { params, headers } from './business';
+import { generateProxyV1Payload } from '../../../testUtils';
+export const v1OauthScenarios = [
+  {
+    id: 'criteo_audience_oauth_0',
+    name: 'criteo_audience',
+    description: '[OAUTH]:: Test expired access token',
+    successCriteria: 'Should return a 401 status code with authErrorCategory as REFRESH_TOKEN',
+    scenario: 'oauth',
+    feature: 'dataDelivery',
+    module: 'destination',
+    version: 'v1',
+    input: {
+      request: {
+        body: generateProxyV1Payload(
+          {
+            JSON: {
+              data: {
+                type: 'ContactlistAmendment',
+                attributes: {
+                  operation: 'add',
+                  identifierType: 'madid',
+                  identifiers: ['sample_madid', 'sample_madid_1', 'sample_madid_2'],
+                  internalIdentifiers: false,
+                },
+              },
+            },
+            params,
+            headers,
+            method: 'PATCH',
+            endpoint: 'https://api.criteo.com/2022-10/audiences/3485/contactlist',
```

Comment:
For `oauth` mocks these mocks needs to added in a separate `platform` folder example [here](https://github.com/rudderlabs/rudder-transformer/blob/9e047747a119cd3e23cb0c352363788f40e0ef42/test/integrations/common/google/network.ts#L4)

Use the following structure

Reply:
Similarly for `critero` oauth scenario we need to create a platform folder `criteo` andadd the scenarios there